src/seedu/addressbook/AddressBook.java

```diff
@@ -552,6 +559,18 @@ private static String executeListAllPersonsInAddressBook() {
         showToUser(toBeDisplayed);
         return getMessageForPersonsDisplayedSummary(toBeDisplayed);
     }
+    
```

Comment:
missing header comments for this method. three things to do when you add a new feature
1. add header comments
2. add help / user guide
3. update test cases for the program